html/apps/applist/index.html

```diff
@@ -71,6 +71,7 @@
                   <div class="hidden">
                     <div class="caption">
                       <div class="centered">
+                        <p><a href='../../apps/{{ app.name }}'>{{ app.headline }}</a></p>
```

Comment:
That wouldn't work because the file is in the applist directory, so you can't access ```/apps/``` directly.

Although you could have written ```../'{{ app.name }}'```.